components/TextImage.vue

```diff
@@ -1,11 +1,25 @@
 <template>
-  <div v-editable="blok">
-    <article>
-      <div>
-        <h2>{{ blok.title }}</h2>
-        <rich-text-renderer :document="blok.description" />
+<<<<<<< HEAD
```

Comment:
Hier is een merge conflict, dit nooit pushen. ik zal dit even fixen. kijk naar je terminal als iets pull. als je merge conflicts zie moet je me even laten weten